content/en/tracing/setup_overview/setup/php.md

```diff
@@ -451,6 +451,24 @@ If no core dump was generated, check the following configurations and change the
 1. Ensure you have a suitable `ulimit` set in your system. You can set it to unlimited: `ulimit -c unlimited`.
 1. If your application runs in a Docker container, changes to `/proc/sys/*` have to be done to the host machine. Contact your system administrator to know the options available to you. If you are able to, try recreating the issue in your testing or staging environments.
 
+### Valgrind trace
+
+Preferably, if you want to give us as many hints as possible, try running the application with valgrind (install it from your package manager). This approach will also work in an unprivileged container (unlike core dumps by default).
+
+   - For CLI this is as simple as executing `USE_ZEND_ALLOC=0 valgrind -- php path/to/script.php`.
+   - When running php-fpm execute `USE_ZEND_ALLOC=0 valgrind --trace-children=yes -- php-fpm -F --fpm-config ...`.
```

Comment:
Just thought of this question: what is the performance hit expected running php-fpm/apache through `valgrind`? I expect that it is not usable in production, correct? In this case should we had a note mentioning this?

Reply:
Yes, that's just for reproducing a couple requests, not flooding it.

Reply:
```suggestion
When running `php-fpm` run:
{{< code-block lang="shell" >}}
 USE_ZEND_ALLOC=0 valgrind --trace-children=yes -- php-fpm -F --fpm-config <MORE_OPTIONS>
 {{< /code-block >}}
```

Reply:
I added a suggested wording in the paragraph above.